test/fixtures/schema/optional.json

```diff
@@ -151,7 +151,7 @@
                                   "content": "application/json"
                                 }
                               },
-                              "content": "{}"
+                              "content": "{\n  \"street\": \"\"\n}"
```

Comment:
Optional without any possible value being rendered. I don't think we should render it in the case it has no values.

```apib
# Group Example
# Resource [/example]

## Action [GET]

+ Response 200 (application/json)

    + Attributes
        - street (optional)
```

Reply:
This may also introduce regressions for dredd users as when an optional property is present in request body it MAY have additional validation. As we're using empty string it might not meet any validation on server side and thus cause existing Dredd to start failing using request bodies generated with this logic.